retis/src/module/skb/bpf/skb_hook.bpf.c

```diff
@@ -1,6 +1,7 @@
 #include <vmlinux.h>
 #include <bpf/bpf_core_read.h>
 #include <bpf/bpf_endian.h>
+#include <if_vlan.h>
```

Comment:
nit: please add this below the inclusion of "common.h", this block is for external ones.